servicekeeper-adapter/servicekeeper-spring-adapter/src/main/java/esa/servicekeeper/adapter/spring/aop/AbstractServiceKeeperAop.java

```diff
@@ -83,6 +79,10 @@ protected void enableServiceKeeper() {
     protected void retry() {
     }
 
+    @Pointcut("@annotation(esa.servicekeeper.core.annotation.Fallback)")
+    protected void fallback() {
```

Comment:
Fix the bug of being proxied many times. See details in:
- `JaxRsAutoSupportAop.doInvoke0`
- `RestlightAutoSupportAop.doInvoke0`
- `DefaultServiceKeeperAop.doInvoke0`
- `WebAutoSupportAop.doInvoke0`